OpenTaiko/src/Common/ImGuiDebugWindow.cs

```diff
@@ -324,6 +324,8 @@ private static void Stage() {
 									}
 								}
 
+								ImGui.Text("Lyrics: " + (dtx.bLyrics ? (dtx.usingLyricsFile ? dtx.listLyric2.Count : dtx.listLyric.Count) : "None"));
```

Comment:
The condition sandwich can be flatten using early-determination.
```suggestion
								ImGui.Text("Lyrics: " + (!dtx.bLyrics ? "None"
								     : dtx.usingLyricsFile ? dtx.listLyric2.Count
								     : dtx.listLyric.Count));
```

Reply:
Or twice.
```suggestion
								    : !dtx.usingLyricsFile ? dtx.listLyric.Count
								    : dtx.listLyric2.Count));
```